homework3/task1/task1.cpp

```diff
@@ -0,0 +1,36 @@
+﻿#include <iostream>
+
+using namespace std;
+
+double powerNumber(double number, double power);
+bool isCurrentPower(double number);
+
+int main()
+{
+	double number, power;
+
+	cout << "Please, enter number: ";
+	cin >> number;
+	cout << "and enter power: ";
+	cin >> power;
```

Comment:
Create function for this code

Reply:
![image](https://user-images.githubusercontent.com/59542254/111069661-2c419100-84df-11eb-80be-cc3c084f9cc8.png)
 done

Reply:
No, create one function instead of two one